code/lib/store/template/stories/rendering.stories.ts

```diff
@@ -65,7 +65,5 @@ export const ChangeArgs = {
 
     await within(canvasElement).findByText(/New Text/);
     await expect(button).toHaveFocus();
-
-    await channel.emit(RESET_STORY_ARGS, { storyId: id });
```

Comment:
resetting args at the end of this test is unnecessary, because we recently reset args at the beginning instead - both as a way to ensure that the story is in a clean state before executing.